MainRepository/Program.cs

```diff
@@ -1,7 +1,10 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
-}
\ No newline at end of file
+﻿// Program to find the given word is palindrome or not.
+Console.Write ("Enter the word or sentence: ");
+string str = Console.ReadLine ().ToLower ().Replace (" ", "");
+string rev = "";
+for (int i = str.Length - 1; i >= 0; i--)
+   rev += str[i];

```

Comment:
Adding string with one character is inefficient. Say you have the word as Assignment. rev will be t in first pass:
rev = t
rev = tn
rev = tne
...
rev = tnemngissa
9 rev strings will be garbage collected as string is immutable each += creates a copy of string. Do it in different way.